ivy/functional/frontends/torch/tensor.py

```diff
@@ -173,6 +173,11 @@ def addmm_(self, mat1, mat2, *, beta=1, alpha=1):
     def addmv(self, mat, vec, *, beta=1, alpha=1):
         return torch_frontend.addmv(self, mat, vec, beta=beta, alpha=alpha)
 
+    @with_unsupported_dtypes({"2.0.1 and below": ("float16",)}, "torch")
+    def addmv_(self, mat, vec, *, beta=1, alpha=1):
+        self.ivy_array = torch_frontend.addmv(self, mat, vec, beta=beta, alpha=alpha)
```

Comment:
Here we need to ensure that we are assigning `self.ivy_array` to correct `ivy.Array` instance
```suggestion
        self.ivy_array = torch_frontend.addmv(self, mat, vec, beta=beta, alpha=alpha).ivy_array
```